tests/test_oteclient.py

```diff
@@ -39,12 +39,15 @@ def test_create_strategies(
     import requests
 
     if strategy == "function":
-        if client._impl._backend == "services" and "example" in client.url:
+        if (
+            getattr(client._impl._backend, "value", client._impl._backend) == "services"
+            and "example" in client.url
+        ):
             pass
         else:
             pytest.skip("No function strategy exists in oteapi-core yet.")
 
-    backend = client._impl._backend
+    backend = getattr(client._impl._backend, "value", client._impl._backend)
```

Comment:
The syntax to figure out what kind backend the client is seems very strange now. Now granted, its hidden attribute of a hidden attribute, but still, it borders on the point where it would seem to be nice to have a some sort of getter method, maybe even figure out how to manually reproduce the original syntax

Reply:
Sure. Again, this is part of a test, not the actual package.